{{ cookiecutter.project_slug }}/{{ cookiecutter.project_module }}/urls.py

```diff
@@ -27,6 +30,19 @@
 except ImportError:
     HAVE_DDT = False
 
+schema_view = get_schema_view(
+   openapi.Info(
+      title='{{ cookiecutter.project_name }}',
+      default_version='v1',
```

Comment:
v1Alpha by default until finished?

Reply:
Possibly. If so, we'd need to update the preferences webapp.